canonical-web-and-design/ubuntu.com
# add new landscape page, remove redirect to old landscape landing page

## Done

- Added new landscape page
- Removed redirect to old landscape website landing page
- Built as per [copy doc](https://docs.google.com/document/d/1Q5KPVOn3yaexgSbkZHmXHmWzT5WG3ocYAPthweMwDz4/edit#)
**Note: Links to /landscape/try and /landscape/features will not work as they have not been built yet**

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/landscape
    - Be sure to test on mobile, tablet and desktop screen sizes
**Note: **

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5143
